index.html

```diff
@@ -34,31 +34,235 @@
     <!-- ─── Headline Section ────────────────────────────────────────── -->
     <section class="headline">
         <div class="personal-info">
-            <h1>I'm Cosmos Glad to see you!</h1>
+            <h1>I'm Cosmos <br> Glad to see you!</h1>
             <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="socail-media">
+                <h3>Let's connect</h3>
+                <div class="social-icon">
+                    <ul>
+                        <li>
+                            <a href="#"><img src="./images/Vector-3.svg" alt="twitter"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Linkedin icon.svg" alt="linked-in"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Vector-1.svg" alt="M"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Vector-2.svg" alt="github"></a>
+                        </li>
+                        <li>
+                            <a href="#"><img src="./images/Vector.svg" alt="sure"></a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- ─── End Headline Section ────────────────────────────────────── -->
+
+    <!-- ─── Portfolio Works ─────────────────────────────────────────── -->
+    <section class="works">
+        <div class="grid-container">
+            <!-- ─── Works Project 1 ─────────────────────────── -->
+            <div class="works-container">
+                <div class="banner">
+                    <img src="./images/Portfolio2.png" class="project-img" alt="project image">
+                </div>
+                <div class="project-content">
+                    <div class="project-title">
+                        <h2>Tonic</h2>
+                    </div>
+                    <div class="outline">
+                        <ul>
+                            <li class="banner-list">Canopy</li>
+                            <span class="dot"></span>
+                            <li class="banner-list dev">Back End Dev</li>
+                            <span class="dot"></span>
+                            <li class="banner-list">2015</li>
+                        </ul>
```

Comment:
[OPTIONAL] - It will be great if we modify this code by putting all the `span` inside a `li` tag. 👍 

For our code to be valid we should not put any tag inside a `<ul>` other than an `<li>`. 👌 that way it will be semantically correct ✔️ 

```html
<ul>
  <li class="banner-list">Canopy</li>
  <li><span class="dot"></span></li>
  <li class="banner-list dev">Back End Dev</li>
  <li><span class="dot"></span></li>
  <li class="banner-list">2015</li>
</ul>
```